### DISCUSSION TITLE
Getting error while trying to install extensions (openvscode in Docker)

### DISCUSSION THREAD

**Discussion by nalmabrcom** [gitpod-io/openvscode-server#213] [Q&A]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
Version: 1.62.0
Commit: 15c32d00a65a5ab0932e0da73f6a6fb5911c2a25
Date: 2021-11-03T18:23:22.622Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36
- OS Version: rhel-8, arch, Windows, openvscode is running in docker

Steps to Reproduce:

1. `docker run -d -it --init -p 3000:3000 gitpod/openvscode-server:insiders` 
2. Go to http://lovalhost:3000
3.  Try to install `Python` or any other extension from the marketplace. 

Actual result:
```

[2021-11-05 14:20:09.990] [window] [error] ["Error: getaddrinfo EAI_AGAIN open-vsx.org","    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:67:26)"]
[2021-11-05 14:20:09.997] [window] [error] getaddrinfo EAI_AGAIN open-vsx.org: Error: getaddrinfo EAI_AGAIN open-vsx.org
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:67:26)

```
    
    
![image](https://user-images.githubusercontent.com/65608274/140517969-6fbc9fbe-bb3f-4bf0-aa95-b2144d3d1f27.png)

**Comment by akosyakov** [comment]
It seems to be a network issue in your env: https://stackoverflow.com/questions/40182121/whats-the-source-of-error-getaddrinfo-eai-again For some reasons server cannot resolve marketplace address as I understand it.
